app/controllers/items_controller.rb

```diff
@@ -1,7 +1,29 @@
 class ItemsController < ApplicationController
 
   def browse
-    @res = $api.query({"facet_num" => 500, "facet_sort" => "term|asc" }).facets
+  end
+
+  def browse_facet
+    # Reverse facet name from url-formatting
+    @browse_facet = params[:facet]
+    if @browse_facet.include?("-")
```

Comment:
through the haze of my headcold, I'm having trouble thinking of facets that would have a `-` in their name. Can you give me an example of one?  I can think of `.` for nested fields and ` and date fields?